litmus-portal/backend/graphql-server/pkg/database/db.go

```diff
@@ -0,0 +1,102 @@
+package database
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/graph/model"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var clusterCollection *mongo.Collection
+
+func DBInit() {
+	dbServer := os.Getenv("DB_SERVER")
+	clientOptions := options.Client().ApplyURI("mongodb://" + dbServer)
+	client, err := mongo.Connect(context.TODO(), clientOptions)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	ctx, _ := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
Put Context.Background() in a struct and reuse it using a receiver to the function